src/ui/server/controllers/insurance/declarations/anti-bribery/index.test.ts

```diff
@@ -72,12 +66,6 @@ describe('controllers/insurance/declarations/anti-bribery', () => {
   });
 
   describe('get', () => {
-    it('should call api.keystone.application.declarations.getLatestAntiBribery', async () => {
```

Comment:
Can we remove the `api.keystone.application.declarations` directory from the UI?

Reply:
done